clientbuffer.cpp

```diff
@@ -52,9 +52,10 @@ class CClientBufferMod : public CModule
     MODCONSTRUCTOR(CClientBufferMod)
     {
         AddHelpCommand();
-        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier>", "Add a client.");
+        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier> [timelimit]", "Add a client with optional time limit.");
```

Comment:
Is there any reason to have two ways to specify the time limit (`AddClient` and `SetClientTimeLimit`)?

Specifying client settings in `AddClient` has the same design issue as `ChangeClient`.

Sometimes, specifying options when creating some object is necessary for security reasons (e.g. `open(fn, 0600)` instead of (`open(fn) + chmod(fn, 0600)`), but I don't see the need here.